ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh

```diff
@@ -185,23 +185,25 @@ for image in "${images[@]}"; do
 done
 
 mkdir -p "${ARTIFACT_DIR}/junit"
+echo "Generating the Junit for fips-or-die scan"
+filename="fips-or-die"
```

Comment:
Could you help add prefix `junit_`, so that it will also be collected into sippy ?

Reply:
you mean in the file name?

Reply:
yes